genie-web/src/main/java/com/netflix/genie/web/configs/aws/AwsS3Config.java

```diff
@@ -100,12 +100,12 @@ public AmazonS3Client genieS3Client(
      * @return An s3 implementation of the FileTransfer interface
      * @throws GenieException if there is any problem
      */
-    @Bean
+    @Bean(name = "s3")
```

Comment:
these names seem kind of risky as they're so short and possibly conflicting with any beans from other boot services. can we use s3:// in there or something? Seems unlikely